src/main/scala/org/constellation/rewards/SoftStaking.scala

```diff
@@ -12,20 +15,25 @@ object SoftStaking extends SoftStaking {
 
   def getAddress: String = address
 
-  def weightBySoftStaking(softStakingNodes: Int)(distribution: Map[String, Double]): Map[String, Double] = {
-    val fullNodes = distribution.size
+  def weightBySoftStaking(
+    ignore: Set[String]
+  )(softStakingNodes: Int)(distribution: Map[String, Double]): Map[String, Double] = {
+    val withoutIgnored = distribution.filterNot { case (address, _) => ignore.contains(address) }
```

Comment:
Can it be implemented just as this - seems like it's semantically the same:
`val withoutIgnored = distribution -- ignore`

Reply:
:+1: